### ISSUE TITLE
Bronchopulmonary dysplasia

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6158]
**Preferred term label:**
Bronchopulmonary dysplasia

**Synonyms**
BPD

**Definition (free text, please give PubMed ID)**
A chronic lung disease most commonly seen in premature infants who required mechanical ventilation and oxygen therapy for acute respiratory distress; PMID: 28067830

**Parent term (use hpo.jax.org/app)**
Abnormal lung morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:70589

**Comment by pnrobinson** [comment]
This is a disease rather than a phenotypic feature. See for instance this ORPHA entry (https://hpo.jax.org/app/browse/disease/ORPHA:70589).
